lib/reline/line_editor.rb

```diff
@@ -969,11 +969,19 @@ def add_dialog_proc(name, p, context = nil)
     end
   end
 
-  private def rerender_added_newline(prompt, prompt_width)
-    scroll_down(1)
+  private def rerender_added_newline(prompt, prompt_width, prompt_list)
     @buffer_of_lines[@previous_line_index] = @line
     @line = @buffer_of_lines[@line_index]
-    unless @in_pasting
+    @previous_line_index = nil
```

Comment:
Why did you move this line?

Reply:
I move it because I am using `whole_lines` in line 979.

In other place of line_editor.rb, whole_lines is called in this form
```ruby
if @previous_line_index
  whole_lines(index: @previous_line_index, line: @line)
else
  whole_lines
end
```
`whole_lines` also depends on `@buffer_of_lines`, `@line` and `@line_index`. Calling `whole_lines` between changing these instance variables seems dangerous for me.
After moving it, `@buffer_of_lines` `@line` `@previous_line_index` is changed all at once.

If I don't move it, it still works but won't work after #460 is merged.

Reply:
I see. I understood. Thank you for your reply.